Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR_description.md

```diff
@@ -16,6 +16,8 @@
 
 ### Mirroring
 
+**XDR mirroring delay in minutes**: In case of missing updates in mirroring incoming changes from XDR, use the xdr_delay parameter to extend the look-back period. However, be aware that this may result in increased latency when updating incidents.
```

Comment:
This is not a look back, this is a delay - there is a difference - look back is a regular fetch/mirroring, but for each time (on time) we are looking extra look back minutes back. in the delay we are delaying the whole process of mirroring to be x minutes ago.